components/observations/ObservationsListView.tsx

```diff
@@ -49,17 +50,25 @@ interface ObservationFragmentWithPageIndexAndZoneAndSource extends ObservationFr
 
 export const ObservationsListView: React.FunctionComponent<ObservationsListViewProps> = ({center_id, requestedTime, initialFilterConfig, ...props}) => {
   const endDate = requestedTimeToUTCDate(requestedTime);
-  const originalFilterConfig: ObservationFilterConfig = {
-    dates: {
-      value: 'past_week',
-    },
-    ...initialFilterConfig,
-  };
+  const originalFilterConfig: ObservationFilterConfig = useMemo(
+    () => ({
+      dates: {
+        value: 'past_week',
+      },
+      ...initialFilterConfig,
+    }),
+    [initialFilterConfig],
+  );
   const [filterConfig, setFilterConfig] = React.useState<ObservationFilterConfig>(originalFilterConfig);
   const [filterModalVisible, setFilterModalVisible] = React.useState<boolean>(false);
   const mapResult = useMapLayer(center_id);
   const mapLayer = mapResult.data;
 
+  // when the initial filter inputs change, we should honor those and try to overlay the user's requests on top of the new base
+  React.useEffect(() => {
+    setFilterConfig(current => _.assign({}, current, originalFilterConfig));
```

Comment:
I don't think you need lodash here? Also the comment is not right I think...or at least the code is not doing what the comment says. It's laying the new base on top of the user's requests, not vice-versa.

```suggestion
    setFilterConfig(current => {
      ...current,
      ...originalFilterConfig
    });
```